mx.sulong/libs/pthread.h

```diff
@@ -0,0 +1,15 @@
+#ifndef _PTHREAD_H
```

Comment:
Okay, I understand. But then we need to use this header file instead of Clib's `pthread.h` header during compilation. Otherwise the unreachable instruction is still created.  To get this to work we have to modify the mx compile commands to include the header files. As an alternative we could also substitute the commands on the Java level and strip the unreachable instruction.